src/llvm/abi/x86.cr

```diff
@@ -1,6 +1,6 @@
 require "../abi"
 
-# Based on https://github.com/rust-lang/rust/blob/master/src/librustc_trans/trans/cabi_x86.rs
+# Based on https://github.com/rust-lang/rust/blob/master/src/librustc_trans/cabi_x86.rs
```

Comment:
Also I think you shouldn't use `master`, but a specific commit hash

Reply:
Changed